core/src/main/java/org/elasticsearch/indices/analysis/PreBuiltTokenFilters.java

```diff
@@ -108,26 +109,24 @@ public TokenStream create(TokenStream tokenStream, Version version) {
 
     STOP(CachingStrategy.LUCENE) {
         @Override
+        @SuppressWarnings("deprecation") // For backwards compatibility
         public TokenStream create(TokenStream tokenStream, Version version) {
             if (version.luceneVersion.onOrAfter(org.apache.lucene.util.Version.LUCENE_4_4_0)) {
                 return new StopFilter(tokenStream, StopAnalyzer.ENGLISH_STOP_WORDS_SET);
             } else {
-                @SuppressWarnings("deprecation")
-                final TokenStream filter = new Lucene43StopFilter(true, tokenStream, StopAnalyzer.ENGLISH_STOP_WORDS_SET);
-                return filter;
+                return new Lucene43StopFilter(true, tokenStream, StopAnalyzer.ENGLISH_STOP_WORDS_SET);
             }
         }
     },
 
     TRIM(CachingStrategy.LUCENE) {
         @Override
+        @SuppressWarnings("deprecation") // For backwards compatibility
         public TokenStream create(TokenStream tokenStream, Version version) {
             if (version.luceneVersion.onOrAfter(org.apache.lucene.util.Version.LUCENE_4_4_0)) {
                 return new TrimFilter(tokenStream);
```

Comment:
No I didn't plan to backport it. It looks like I can drop anything in master that checks for Lucene version less than 4.6.1. Sound right to you?